src/Appwrite/Utopia/Response/Model/Attribute.php

```diff
@@ -10,6 +10,12 @@ class Attribute extends Model
     public function __construct()
     {
         $this
+            ->addRule('$collection', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Collection ID.',
+                'default' => '',
+                'example' => '',
```

Comment:
we use `'example' => '',` for docs examples. You can just pick a random Appwrite "style" ID.